cmd/internal/cli/action_flags.go

```diff
@@ -196,13 +196,24 @@ var actionShellFlag = cmdline.Flag{
 	Tag:          "<path>",
 }
 
+// --cwd
+var actionCwdFlag = cmdline.Flag{
+	ID:           "actionCwdFlag",
+	Value:        &cwdPath,
+	DefaultValue: "",
+	Name:         "cwd",
+	Usage:        "initial working directory for payload process inside the container (synonym for --pwd)",
+	EnvKeys:      []string{"PWD", "TARGET_PWD"},
+	Tag:          "<path>",
+}
+
 // --pwd
 var actionPwdFlag = cmdline.Flag{
 	ID:           "actionPwdFlag",
-	Value:        &pwdPath,
+	Value:        &cwdPath,
 	DefaultValue: "",
 	Name:         "pwd",
-	Usage:        "initial working directory for payload process inside the container",
+	Usage:        "initial working directory for payload process inside the container (synonym for --cwd)",
```

Comment:
Let's make this one Hidden. Otherwise the usage message will show both `--cwd` and `--pwd` AFAIK.